core/src/navigation/TopNav.svelte

```diff
@@ -855,13 +861,6 @@
     }
   }
 
-  img {
-    &.fd-top-nav__icon {
-      height: 16px;
-      min-width: auto;
-    }
-  }
```

Comment:
Are we sure this is not being used ? Will also run & test but thought maybe its removed by mistake. 🤔

Reply:
I think I removed it because a) Svelte marked it as unused, based on the local checks within this component, so it was not included in the compiled code anyway and b) I found no indication in code that there can be images with that class.. That is why I did not wrap it into a `:global` rule like in other cases. But thanks for double-checking, I may have overlooked something!